tidy3d/web/api/container.py

```diff
@@ -718,6 +735,10 @@ def pbar_description(task_name: str, status: str) -> str:
                 pbar_tasks = {}
                 for task_name, job in self.jobs.items():
                     status = job.status
+                    if config_end_statuses is not None:
+                        run_statuses = config_end_statuses
+                    if config_run_statuses is not None:
+                        end_statuses = config_run_statuses
```

Comment:
it seems to me that these are mixed up? it defines `end_statuses` as `config_run_statuses` and `run_statuses` as `config_end_statuses`? Am I missing something?

Reply:
Modified.